zgw/common-ground-authentication/src/main/kotlin/nl/nlportal/commonground/authentication/AuthenticationMachtigingsDienstService.kt

```diff
@@ -36,7 +36,7 @@ class AuthenticationMachtigingsDienstService(
                 this.authenticationMachtingDiensten = Mapper.get().readValue(json, object : TypeReference<List<AuthenticationMachtigingsDienst>>() {})
             }
         } catch (ex: Exception) {
-            logger.warn("Could not load json from {}", authenticationMachtingsDienstConfig.resourceUrl)
+            logger.warn { "Could not load json from $authenticationMachtingsDienstConfig.resourceUrl with reason ${ex.message}" }
```

Comment:
interpolation doesn't include resourceUrl

```suggestion
            logger.warn { "Could not load json from ${authenticationMachtingsDienstConfig.resourceUrl} with reason ${ex.message}" }
```

Reply:
Will change it